presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.product.launcher.cli;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import com.github.dockerjava.api.command.InspectContainerResponse.ContainerState;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Module;
+import io.airlift.airline.Arguments;
+import io.airlift.airline.Command;
+import io.airlift.airline.Option;
+import io.airlift.log.Logger;
+import io.prestosql.tests.product.launcher.Extensions;
+import io.prestosql.tests.product.launcher.LauncherModule;
+import io.prestosql.tests.product.launcher.PathResolver;
+import io.prestosql.tests.product.launcher.env.Environment;
+import io.prestosql.tests.product.launcher.env.EnvironmentModule;
+import io.prestosql.tests.product.launcher.env.EnvironmentOptions;
+import io.prestosql.tests.product.launcher.env.Environments;
+import io.prestosql.tests.product.launcher.env.SelectedEnvironmentProvider;
+import org.testcontainers.containers.Container;
+
+import javax.inject.Inject;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.net.Socket;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.google.common.base.Preconditions.checkState;
+import static io.prestosql.tests.product.launcher.cli.Commands.runCommand;
+import static java.util.Objects.requireNonNull;
+import static org.testcontainers.containers.BindMode.READ_ONLY;
+
+@Command(name = "run", description = "Presto product test launcher")
+public final class TestRun
```

Comment:
TemptoTestRun?

Reply:
I want to name the class consistently with the command/subcommand it's bound to.